platforms/hyperledger-besu/charts/tessera_key_mgmt/values.yaml

```diff
@@ -25,6 +25,7 @@ image:
   #Provide the image pull secret of image
   #Eg. pullSecret: regcred
   pullSecret:
+  alpineutils:
```

Comment:
Please mention the description of this key. (Because the name can be misleading, pure alpine image wont have the necessary binaries) You can write what this image is for, and where they can find the dockerfile of this for reference (If the user decides to use a different `network.docker.url`)